index.html

```diff
@@ -2,9 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <link rel="icon" href="/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <title>Sadie Crawford, Developer</title>
+    <link rel="icon" href="/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <style>
```

Comment:
It can be a good habit to inline styles you'll want for every page to prevent FOUC (flash of unstyled content) while the browser is resolving your stylesheets.

Reply:
Great to know, ty!! 😄